ops/terraform/modules/resources/asg/main.tf

```diff
@@ -46,12 +46,7 @@ resource "aws_security_group" "base" {
   vpc_id        = var.env_config.vpc_id
   tags          = merge({Name="bfd-${var.env_config.env}-${var.role}-base"}, local.tags)
 
-  ingress {
-    from_port   = 22
-    to_port     = 22
-    protocol    = "tcp"
-    cidr_blocks = ["10.0.0.0/8"]
-  }
+  # Note: If we want to allow Jenkins to SSH into boxes, that would go here.
```

Comment:
Does Jenkins use Ansible? or is Ansible running on a different port?

Reply:
Jenkins fires up packer, which makes use of ansible (run from jenkins) and the temp packer instances are assigned to the `packer_sg` SG group which allows port 22 from Jenkins.

Reply:
So far, Jenkins doesn't directly use Ansible in the CCS. It instead uses Packer to build pre-configured AMIs (or AMIs that will configure themselves using Ansible in a `user_data` script).